src/middlewares/validateBody/schemas/gameSchemas.ts

```diff
@@ -0,0 +1,10 @@
+import { IWord } from '../../../interfaces/GameInterfaces';
+import { JSONSchemaType } from 'ajv';
+
+export const userSchema: JSONSchemaType<IWord> = {
```

Comment:
should this export be `userSchema` or `gameSchema`?
Also, maybe file should be named gameSchema.ts, not gameSchema**s**.ts to be consistent with other schema files

Reply:
Thank you ! I updated that, as probably there are going to be more schemas related to the game, I changed the file and export name to gameWordSchema. 

In future, we might create a separate folder for storing all these related schemas.